File: Lancetilla FrontEnd/src/app/demo/Models/CuidadoPlantasViewModel.ts
export class CuidadoPlantasViewModel {
    constructor(
    public cupl_Id?: number,
    public plan_Id?: number,
    public plan_Codigo?:string,
    public tipl_Id?: number,
    public arbo_Id?: number,
    public tipl_NombreComun?: string,
    public tipl_NombreCientifico?: string,
    public ticu_Id?: number,
    public ticu_Descripcion?: string,
    public cupl_Fecha?: string,
    public cupl_UserCreacion?: number,
    public cupl_FechaCreacion?: string,
    public usua_UserCreaNombre?:string,
     public usua_UserModiNombre?: string,
     public cupl_Estado?: boolean
    ){}
}

File: Lancetilla BackEnd/Lancetilla/Lancetilla.API/Models/CuidadoPlantasViewModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Lancetilla.API.Models
{
    public class CuidadoPlantasViewModel
    {

        public int cupl_Id { get; set; }
        public int plan_Id { get; set; }
        public int ticu_Id { get; set; }

        public int arbo_Id { get; set; }

        public string cupl_Fecha { get; set; }
        public int? cupl_UserCreacion { get; set; }
        public int? cupl_UserModificacion { get; set; }

    }
}
